src/UrlTracker.Backoffice.UI/package.json

```diff
@@ -8,36 +8,30 @@
     "build": "tsc && vite build",
     "build:watch": "vite build --watch",
     "preview": "vite preview",
-    "lint": "eslint . --ext .ts,.tsx"
+    "lint": "eslint . --ext .ts,.tsx",
+    "format": "prettier --write ."
   },
   "dependencies": {
-    "@lit/context": "^1.0.0",
-    "@oddbird/popover-polyfill": "^0.2.0",
-    "@umbraco-ui/uui": "^1.4.0",
-    "axios": "^1.5.1",
-    "chart.js": "^4.4.2",
+    "@lit/context": "^1.1.1",
+    "@oddbird/popover-polyfill": "^0.4.3",
+    "@umbraco-ui/uui": "^1.8.1",
+    "axios": "^1.7.2",
+    "chart.js": "^4.4.3",
     "js-cookie": "^3.0.5",
-    "lit": "^3.0.0",
+    "lit": "^3.1.3",
     "qs": "^6.12.1"
   },
   "devDependencies": {
-    "@types/angular": "^1.8.6",
+    "@types/angular": "^1.8.9",
```

Comment:
So I checked out the Umbraco source code for Umbraco 13 and found the package.json that Umbraco uses in U13:
https://github.com/umbraco/Umbraco-CMS/blob/v13/dev/src/Umbraco.Web.UI.Client/package.json

It looks like they use uui version 1.7.1 and angular version 1.8.3.
So they should be updated, but I think we should match the versions that Umbraco uses to prevent issues due to version mismatches.